zivid_samples/src/sample_capture.cpp

```diff
@@ -46,26 +46,35 @@ int main(int argc, char** argv)
   ros::AsyncSpinner spinner(1);
   spinner.start();
 
-  auto points_sub = n.subscribe("/zivid_camera/points", 1, on_points);
+  auto points_sub = n.subscribe("/zivid_camera/points/xyzrgba", 1, on_points);
 
-  ROS_INFO("Enabling the reflection filter");
-  dynamic_reconfigure::Client<zivid_camera::CaptureGeneralConfig> capture_general_client("/zivid_camera/"
-                                                                                         "capture/general/");
-  zivid_camera::CaptureGeneralConfig config;
-  CHECK(capture_general_client.getDefaultConfiguration(config, default_wait_duration));
-  config.filters_reflection_enabled = true;
-  CHECK(capture_general_client.setConfiguration(config));
+  dynamic_reconfigure::Client<zivid_camera::SettingsConfig> settings_client("/zivid_camera/"
```

Comment:
It seems a bit involved to use the sample. What if I only care about the pointcloud, and not about how the dynamic reconfigure setup works, and just want to `capture(Zivid::Settings settings)`?
(I guess this could be done similar to the `CaptureSettingsController<...>::setZividSettings(const ZividSettingsType& settings)`).

- I guess the reason the settings cannot be passed in this way is that it should also work for Python? And perhaps the interaction with the Capture Assistant's suggested settings? 

An alternative could perhaps be to have some sort of wrapper function in `zivid-sample`? 

HDR captures could be quite verbose here, but that might be something we'll have to accept. Could be interesting to have an [example which captures with settings from yml](https://github.com/zivid/zivid-cpp-samples/blob/master/source/Camera/Basic/CaptureWithSettingsFromYML/CaptureWithSettingsFromYML.cpp). But maybe we should consider that when handling #31.

Reply:
From the sample you must set the settings via dynamic_reconfigure (at least one acquisition). I agree that it becomes cumbersome, it is just the way it is now. There is no way to use Zivid::Settings or provide a YML from the clients code. We could maybe have a command that takes a YML file and applies it.

Reply:
I added an issue for this in #33. You can add there if you have any thoughts.